File: js/beersong.js
function beersong (count) {

if (count > 1) {

var word = "bottles";

    console.log(count + " " + word + " of beer on the wall." + count + " " + word + " of beer,");
    console.log(count + " " + word + " of beer,");
    console.log("Take one down, pass it around,");
    count = count - 1;
    beersong(count);
  }
else {     
        var word = "bottle";
        console.log(count + " " + word + " of beer on the wall.");
        console.log("No more " + word + " of beer on the wall.\nWe've taken them down and passed them around; now we're drunk and passed out!");        
      }
}
beersong(99);

File: c/beersong.c
#include "stdio.h"



  void beerSong(int count) {

  printf("%d bottles of beer on the wall. %d bottles,\n", count, count);
  printf("%d bottles of beer,\n", count);
  printf("Take one down, pass it around,");

  if (count=1) {    
        
        printf("%d bottle of beer on the wall.\n");
        printf("No more bottles of beer on the wall.");   

  } else {
    printf("%d bottles of beer on the wall.\n", count-1);
    beerSong(count-1);
  }

  }
  int main(){

  beerSong(99);
  return 0;
}